src/main/resources/org/jenkinsci/plugins/envinject/EnvInjectBuildWrapper/config.jelly

```diff
@@ -35,6 +21,20 @@
                 value="${instance.info.groovyScriptContent}"/>
     </f:entry>
 
+    <f:entry field="propertiesFilePath"
```

Comment:
NIT: "Properties content" is the most widely used section, so I would use another direction. BTW, the interface should be reworked to show only the required section (extension point?). Such rework is not a part of the PR, so I'd prefer to submit it as another PR